docs/spec/openapi.yml

```diff
@@ -579,6 +579,48 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  '/namespaces/{namespace}/database-engines/{name}/upgrade-operator':
```

Comment:
It doesn't feel natural for the REST API to have a verb in the path. Wdyt about using 
put `/namespaces/{namespace}/database-engines/{name}/operator`?

The meaning remains the same. In the path we have the object we work with - the operator, in the method we have the verb about what action is performed over it - PUT.

Reply:
Hmm, I agree about the verb in the path. However `PUT /namespaces/{namespace}/database-engines/{name}/operator` also doesn't seem to indicate that it is meant to "upgrade" the operator version.

How about `PUT /namespaces/{namespace}/database-engines/{name}/operator/version`

Reply:
I see your concern. I'm wondering if we're going to have any other actions over the `/operator` and why do we need it as a separate segment in the path. However we want to specify we update _the version_ and not anything else. Hmmm... 
What about `PUT /namespaces/{namespace}/database-engines/{name}/operator-version` then?